ParisBNF/abb/BNFabb142.xml

```diff
@@ -0,0 +1,192 @@
+                     </layoutDesc>
+
+                  </objectDesc>
+                  <handDesc>
+                   <handNote xml:id="h1" script="Ethiopic">
+                        <seg type="script">First half of the 19th century.</seg>
+                             <seg type="ink">Black, red.</seg>
+                             <desc>Crude, poor handwriting.</desc>
+                             </handNote>
+                  </handDesc>
+
+                  <bindingDesc>
+                     <binding contemporary="Ethiopian" cert="medium">
+                        <decoNote xml:id="b1">One board, remains of sewing.</decoNote>
+
+                  <decoNote xml:id="b2" type="SlipCase">The manuscript has a poorly preserved leather case.</decoNote>
+                              <decoNote xml:id="b8" type="bindingMaterial">
+                                    <material key="wood"/>
+                              </decoNote>
+
+                    </binding>
+                  </bindingDesc>
+               </physDesc>
+               <history>
+                  <origin>
+                     <origDate notBefore="1800" notAfter="1848">The first half of the 19th century.</origDate>
+                     One of the owners was <persName ref="PRS13891KenfaMikael" role="owner"/>.</origin>
+                    <provenance>The manuscript was possibly in possession of <persName ref="PRS13441WaldaYohannes" cert="medium"/> before it was
+                      acquired by <persName ref="PRS1071dAbbadi" role="owner">Antoine d'Abbadie</persName>
+                      sometime between <date>1840</date> and <date>1848</date>.
+                    </provenance>
+                    <acquisition>The manuscript entered <placeName ref="INS0303BNF"/> in
+                      <date>1902</date>.
+                </acquisition>
+               </history>
+               <additional>
+                  <adminInfo>
+                     <recordHist>
+                        <source>
+                           <listBibl type="catalogue">
+                              <bibl>
+
+                                            <ptr target="bm:ContiRossini1914Notice"/>
+                                            <citedRange unit="page">71-72</citedRange>
+                                            <citedRange unit="number">47</citedRange>
+                                         </bibl>
+                                         <bibl>
+                                            <ptr target="bm:Chaine1912Catalogue"/>
+                                            <citedRange unit="page">88</citedRange>
+                                            <citedRange unit="number">142</citedRange>
+                                         </bibl>
+                                         <bibl>
+                                            <ptr target="bm:Abbadie1859Cat"/>
+                                            <citedRange unit="page">141</citedRange>
```

Comment:
page and number are reversed and not exact 
page 157-158, number 142, not 141

Reply:
done